app/main/views/suppliers.py

```diff
@@ -866,6 +868,48 @@ def toggle_supplier_services(supplier_id):
     return redirect(url_for('.find_supplier_services', supplier_id=supplier_id))
 
 
+def _draft_services_annotated_unanswered_counts(framework_slug, draft_services):
+    try:
+        manifest = content_loader.get_manifest(framework_slug, "edit_service_as_admin")
+    except ContentNotFoundError:
+        return None
+
+    return tuple(
+        {
+            **draft_service,
+            "unansweredRequiredCount": count_unanswered_questions(
+                manifest.filter(draft_service).summary(draft_service)
+            )[0],
+        } for draft_service in draft_services
+    )
+
+
+@main.route('/suppliers/<int:supplier_id>/draft-services', methods=['GET'])
+@role_required('admin-framework-manager')
```

Comment:
I think we should probably add the Sourcing role here too (not sure about Category admins, they generally care about live services rather than drafts, but it's easy enough to add them in later if needed).

Can we add a quick test to check that the other roles 403 on this view? We have that test pattern for most of the other admin views:

```
@pytest.mark.parametrize("role,expected_code", [
        ("admin", 403),
        ("admin-ccs-category", 403),
        ("admin-ccs-sourcing", 200),
        ("admin-manager", 403),
        ("admin-framework-manager", 200),
    ])
...
```

Reply:
Righto. Should sourcing also be able to see "all" framework statuses?

Reply:
Yes.

Reply:
great - done